tox.ini

```diff
@@ -32,10 +32,11 @@ setenv =
     pip19.1: PIP==19.1
 
     coverage: PYTEST_ADDOPTS=--strict --doctest-modules --cov --cov-report=term-missing {env:PYTEST_ADDOPTS:}
-install_command = python -m pip install -U {opts} {packages}
-commands =
+install_command = python -m pip install {opts} {packages}
```

Comment:
That's the default now, so can be removed.

Reply:
Right!